resources/sts/4.11/enforced/sts_installer_permission_policy_1.json

```diff
@@ -0,0 +1,172 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
```

Comment:
I'm assuming this list isn't condition based for a reason. I mostly see ELB and S3 related stuff in here that might need to be condition-ized

Reply:
S3 is restricted by bucket via resource and the ELB's doesn't support condition keys.